.github/workflows/bump_torch_mlir.yml

```diff
@@ -48,12 +48,13 @@ jobs:
         id: bump_submodule
         run: |
           cd third_party/torch-mlir
-          git fetch origin
-          if [ "${{ github.event_name }}" == "workflow_dispatch" ]; then
-            git checkout ${{ github.event.inputs.bump_target }}
+          if [ "workflow_dispatch" == "workflow_dispatch" ]; then
+            bump_target=${{ github.event.inputs.bump_target }}
           else
-            git checkout origin/main
+            bump_target=origin/main
           fi
+          git fetch ${bump_target//\// }
```

Comment:
Why the `//\//`?

Reply:
substitute the `/` with a space bcs bump target is specified as remote/branch whereas fetch requires `remote branch`

Reply:
should this be a comment?

Reply:
I'm confused... is this workflow ever going to fetch a remote other than `origin`? I guess you're trying to support branches, tags, and commit hashes?